runners/core-java/src/test/java/org/apache/beam/runners/core/SimpleDoFnRunnerTest.java

```diff
@@ -386,6 +399,180 @@ public void testInfiniteSkew() {
             BoundedWindow.TIMESTAMP_MAX_VALUE));
   }
 
+  @Test
+  @Category({ValidatesRunner.class})
```

Comment:
Need to tag with `UsesTimersInParDo.class` and/or `UsesTimerMap.class`.

Reply:
Done. Assuming just the validates runner tests need this because I don't see it elsewhere in the file. Let me know if not.